src/Annotation/ApiResource.php

```diff
@@ -38,6 +38,11 @@
      */
     public $iri;
 
+    /**
+     * @var string
+     */
+    public $routePrefix;
```

Comment:
Do we really need a new attribute for that? Can't we use an attribute?

Reply:
Speaking about that, maybe can we just forward any top-level attribute passed to the annotation to attributes? This way it will be easier for the user. Anyway if we create a new attribute, the XML and the YAML loader must be extracted too (it's why I'm 👎).

Reply:
Well to be honest, I have no idea what qualifies for a top-level attribute and what can be part of `attributes` :-D

Reply:
@dunglas we should totally freeze current top-level stuff and only allow updates with optional attributes IMO

Reply:
Yeah I found it strange that I need to update all the factories. So I'm moving this to attributes?

Reply:
Yes please, I also think it's better to use attributes.

@soyuka what do you think about the fallback mechanism I was mentioning earlier?